api/v1beta1/awscluster_webhook.go

```diff
@@ -114,6 +114,12 @@ func (r *AWSCluster) ValidateUpdate(old runtime.Object) error {
 					r.Spec.ControlPlaneLoadBalancer.Name, "field is immutable"),
 			)
 		}
+		if oldC.Spec.ControlPlaneLoadBalancer.HealthCheckProtocol.String() != ClassicELBProtocolSSL.String() && newLoadBalancer.HealthCheckProtocol != nil {
```

Comment:
If both `oldC.Spec.ControlPlaneLoadBalancer.HealthCheckProtocol` and`newLoadBalancer.HealthCheckProtocol` is TCP, then update will fail? It shouldn't.

Should add this test.

Reply:
Done